optimum/exporters/onnx/model_configs.py

```diff
@@ -567,7 +568,14 @@ class ConvNextOnnxConfig(ViTOnnxConfig):
 
 
 class MobileViTOnnxConfig(ViTOnnxConfig):
-    pass
+    ATOL_FOR_VALIDATION = 1e-4
+
+    @property
+    def outputs(self) -> Dict[str, Dict[int, str]]:
+        if self.task == "image-segmentation":
+            return copy.deepcopy(self._TASK_TO_COMMON_OUTPUTS["semantic-segmentation"])
```

Comment:
Are `semantic-segmentation` a synonym to `image-segmentation`?

If so I would rather use the "synonyms" system we have in the `TasksManager`.

Reply:
I don't think they are. The thing is that `AutoModelForImageSegmentation` will be deprecated: https://github.com/huggingface/transformers/blob/892399c5ffdc3c7d0e4f9b2c1c81511293c1581b/src/transformers/models/auto/modeling_auto.py#L487
So we'll have `AutoModelForSemanticSegmentation`, `AutoModelForUniversalSegmentation` and `AutoModelForInstanceSegmentation`.
Then, I don't know what the project regarding the `image-segmentation` pipeline task in Transformers is moving forward, I'm going to ask.

Reply:
Alright. Do not want to complicate things, we can merge, but let's add a comment explaining this in the code so we understand in the future.

Reply:
Another solution is to remove the `"pred_boxes"` and `"pred_masks"` outputs of "image-segmentation": https://github.com/huggingface/optimum/blob/1a5443b8ed8cd4ce1375055817ae6e75926939a6/optimum/exporters/onnx/base.py#L136
I don't remember why I put them when I added this. But this won't be backwards compatible.

Reply:
Are they useful for an `image-segmentation` task?
I think I am the one who added that comment, or at least we already had the discussionm dont remember lol!

Reply:
Only `pred_masks` is useful for segmentation. I just pushed a new commit to refine that.